spec/interactors/developers/approvals/create_approval_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.describe Developers::Approvals::CreateApproval, type: :interactor do
+  subject(:context) { described_class.call(developer: developer, params: { user: user.email }) }
```

Comment:
can we please use another name for subject?
so we won't be confised with spec contexts